src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
```

Comment:
Stylistically, I'm not a fan of `from X import Y`. Even much less of a fan of `from X import *`. I'd rather have `import X` and write out explicitly in the code. Mostly a nitpick.